internal/pkg/skuba/deployments/ssh/dex.go

```diff
@@ -58,3 +59,20 @@ func dexDeploy(t *Target, data interface{}) error {
 	_, _, err = t.ssh("kubectl --kubeconfig=/etc/kubernetes/admin.conf apply -f /tmp/dex.d")
 	return err
 }
+
+func dexRenewCertificate(t *Target, data interface{}) error {
+	_, _, err := t.ssh("sudo kubectl --kubeconfig=/etc/kubernetes/admin.conf delete secret oidc-dex-cert --namespace=kube-system")
```

Comment:
Like we discussed, please use clientset to do all the operation instead of ssh, because you already have clientset.